thoth-app/src/component/admin.rs

```diff
@@ -1,10 +1,11 @@
+use thoth_api::account::model::AccountDetails;
```

Comment:
Is it OK to borrow the API's data structure like this, or would it be better to create an app-specific version (as we have done in other similar cases)?

Reply:
We're currently doing both. I think in the long run we want to reuse as many structs from the API as possible, though there are many examples in which that is not possible